src/utils/apiEndpoint.js

```diff
@@ -10,7 +10,7 @@ const getApiEndpoint = (location) => {
       return 'http://localhost:3000';
     }
 
-    return 'https://api.scp.biomage.net';
+    return `https://api.${process.env.DOMAIN_NAME}`;
```

Comment:
Does that work in staging? Have you checked that `process.env.DOMAIN_NAME` is defined correctly?

Reply:
yes, I checked in lens and the pod config i can see that DOMAIN_NAME in staging is - scp-staging.biomage.net and production - scp.biomage.net